pages/report-repair/[route].js

```diff
@@ -93,6 +95,25 @@ function ReportRepair() {
           values={values}
         />
       )
+    case 'repair-kitchen-problems':
+      return (
+        <RepairProblemRelatedTo
+          handleChange={handleChange}
+          values={values}
+          options = {[{ value: 'cupboards', title: 'Cupboards, including damaged cupboard doors'}]}
```

Comment:
Only one option for now?
I'm assuming the rest will be covered later on right?

Reply:
Yes, that's correct.